### PR TITLE
Simplify some lines in time_sequence_prediction

### PR DESCRIPTION
**Pull request by momohatt**
This PR simplifies some lines in `time_sequence_prediction/train.py`.
* The variable `i` in line 25 is unused.
* The combination of `torch.stack` and `torch.squeeze` in line 35 can be replaced with a `torch.cat`.
* `input.chunk(input.size(1), dim=1)` in line 25 is identical to `input.split(1, dim=1)`.

The last 2 changes might not be necessary, so please let me know if you want me to undo them. Thanks!

### CODE REVIEW

**Comment by soumith** [comment]
this is great, thanks!
